### PR TITLE
Auto Pandora One

### PR DESCRIPTION
**Pull request by JasonLG1979**
This will automatically set the client free or Pandora One. It removes the need for the Pandora One Checkbox.

### CODE REVIEW

**Comment by JasonLG1979** [comment]
If you don't have a Pandora One account @TingPing I'll email you my creds for my main account so you can test this.

**Comment by JasonLG1979** [comment]
This also has the benefit that it doesn't require any sort of migrating of settings.

**Comment by TingPing** [comment]
I don't but I'll go on good faith you tested it throughly enough as its a fairly small patch.

File: pithos/PreferencesPithosDialog.py [hunk 1]
```diff
@@ -213,11 +210,10 @@ def cb(success):
...
             email = self.email_entry.get_text()
             password = self.password_entry.get_text()
-            pandora_one = self.pandora_one_checkbutton.get_active()
 
-            if self.last_email != email or self.last_password != password or self.last_pandora_one != pandora_one:
+            if self.last_email != email or self.last_password != password:
                 SecretService.set_account_password(self.last_email, email, password, cb)
             else:
-                self.settings.revert()
+                self.settings.apply()
```
Comment (TingPing) [inline pithos/PreferencesPithosDialog.py]: What was this change for?

Comment (JasonLG1979) [inline pithos/PreferencesPithosDialog.py]: That's a fix from the last bit. Otherwise just changing the audio quality setting has no effect.

**Pull request merged**